src/libsync/creds/abstractcredentials.h

```diff
@@ -33,6 +33,13 @@ class OWNCLOUDSYNC_EXPORT AbstractCredentials : public QObject
     Q_OBJECT
 
 public:
+    enum class ReadyState {
+        Ready,
+        Retry,
```

Comment:
Maybe Error and FatalError?
Retry implies that we already decided to rety, but thats a user choice.

Reply:
This differentiation already shows how bad the naming is, and also highlights that this change doesn't really make sense in this context. I mean, what kind of error can be so fatal that we can't restart the `OAuth` process and actually need to sign out the user?

Reply:
True I mean with basic out you should also be able to retry